src/scripts/git_data.py

```diff
@@ -13,6 +17,36 @@ def get_last_commit_date(file_path):
         print(f"An error occurred: {e}")
     return None
 
+def get_last_commit_date(file_path):
+    try:
+        # get the last commit date as "September 12, 2022"
+        GITHUB_TOKEN = os.environ['GITHUB_TOKEN']
```

Comment:
This way it has the chance of returning None. Otherwise it raises an exception.
```suggestion
        GITHUB_TOKEN = os.getenv('GITHUB_TOKEN')
```